core/helpers/hardhat.ts

```diff
@@ -0,0 +1,81 @@
+import { getContractAddressByName } from '../src/contracts';
+import BigNumber from '../src/bignumber';
+import { EthereumProvider } from 'hardhat/types';
+import { ethers } from 'ethers';
+import { formatToHex } from './format';
+import { pad32, concat, stripZeros } from './hex';
+import hre from 'hardhat';
+import { setupRpcUrlAndGetNetworks } from '../src/rpc';
+import { createWalletFromPrivateKey } from '../src/signer';
+import { NETWORK, REMOTE_RPC_URL } from '../helpers/constants';
+
+export const overwriteValueInSlot = async (
+    network: string,
+    provider: EthereumProvider,
+    contractName: string,
+    slotAddress: string,
+    newValue: string
+) => {
+    const contractAddress = await getContractAddressByName(network, contractName);
+    const storageToWrite = [contractAddress, slotAddress, newValue];
+    await provider.send('hardhat_setStorageAt', storageToWrite);
+};
+
+export const generateMappingSlotAddress = (mappingStartSlot: string, key: string) => {
+    return stripZeros(ethers.utils.keccak256(concat(pad32(key), pad32(mappingStartSlot))));
+};
+
+export const overwriteUintValue = async (
+    network: string,
+    provider: EthereumProvider,
+    contractName: string,
+    slotAddress: string,
+    value: BigNumber
+) => {
+    const newValue = formatToHex(value, 32);
+    await overwriteValueInSlot(network, provider, contractName, slotAddress, newValue);
+};
+
+export const overwriteUintMapping = async (
+    network: string,
+    provider: EthereumProvider,
+    contractName: string,
+    mappingStartSlot: string,
+    key: string,
```

Comment:
Continuing the discussion in https://github.com/sidestream-tech/unified-auctions-ui/pull/408#discussion_r949008033, maybe at least we can name arguments like that to keep them consistent? And possibly provide docstring with the example? I am not sure anyone except you will understand what needs to be provided here
```suggestion
    mappingSlotAddress: string,
    mappingKey: string,
```